lib/customer_io.dart

```diff
@@ -1,8 +1,18 @@
-
+import 'customer_io_config.dart';
 import 'customer_io_platform_interface.dart';
 
 class CustomerIo {
-  Future<String?> getPlatformVersion() {
-    return CustomerIoPlatform.instance.getPlatformVersion();
+  const CustomerIo._();
```

Comment:
🤔 What is this for? Curious....

Reply:
This is how you make `singleton` classes in dart